content/docs/lifting-state-up.md

```diff
@@ -110,15 +110,15 @@ class Calculator extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/jGBryx?editors=0010)
+[**Cobalah di CodePen**](https://codepen.io/gaearon/pen/jGBryx?editors=0010)
 
-We have two inputs now, but when you enter the temperature in one of them, the other doesn't update. This contradicts our requirement: we want to keep them in sync.
+Kami memiliki dua input sekarang, tetapi ketika anda memasukkan suhu di salah satu dari mereka, yang lain tidak memperbarui. Ini bertentangan dengan persyaratan kami: kami ingin tetap menyinkronkannya.
 
-We also can't display the `BoilingVerdict` from `Calculator`. The `Calculator` doesn't know the current temperature because it is hidden inside the `TemperatureInput`.
+Kami juga tidak dapat menampilkan `BoilingVerdict` dari `Calculator`. `Calculator` tidak tahu suhu saat ini karena tersembunyi di dalam `TemperatureInput`.
```

Comment:
`We also can't display` sepertinya `Kita juga tidak dapat menampilkan`

Reply:
`` `Calculator` tidak tahu suhu saat ini karena suhu tersebut tersembunyi di dalam `TemperatureInput`.``